suite-common/wallet-utils/src/fiatConverterUtils.ts

```diff
@@ -11,27 +11,21 @@ export const toFiatCurrency = (
     decimals = 2,
 ) => {
     // calculate amount in local currency
-
     const rate = networkRates?.[fiatCurrency];
     if (!rate) {
         return null;
     }
 
-    let formattedAmount = amount;
-    if (typeof amount === 'string') {
```

Comment:
Are you 100% sure that this won't break anything? For example something in desktop app?

Reply:
TLDR: Yes.

- amount parameter is typed as string, so unless it's used somewhere with @ts-ignore and different value type, getting rid of this check is not going to result in a bug
<img width="578" alt="image" src="https://user-images.githubusercontent.com/36101761/196584400-db956fb2-f48b-4d43-a37f-76801960c6ef.png">
- after getting rid of the check, things look simpler and it's very obvious that there is pointless reassignment into new variable just as the IDE points out 
<img width="607" alt="image" src="https://user-images.githubusercontent.com/36101761/196584665-c06e3390-7937-4e90-84c0-73becc49dfa6.png">

So I am pretty sure it shouldn't break anything. Do you see something weird about this refactor, or are you just making sure? Looks alright to me and works as expected. 

There are also unit tests for this file which are still passing.

Reply:
Cool.